doc/fzf-vim.txt

```diff
@@ -203,7 +203,7 @@ A few commands in fzf.vim can be customized with global option variables shown
 below.
 
                *g:fzf_buffers_jump* *g:fzf_commits_log_options* *g:fzf_tags_command*
-                                                         *g:fzf_commands_expect*
+                                    *g:fzf_commands_expect* *g:fzf_git_auto_chdir*
```

Comment:
I'm thinking, we should probably make this behavior the default, without an additional configuration variable. And we can wait and see if anyone complains.

Reply:
:+1: I've removed the commit that added this config.